lib/project-data.ts

```diff
@@ -70,7 +80,7 @@ export class ProjectData implements IProjectData {
 					this.projectFilePath = projectFilePath;
 					this.appDirectoryPath = path.join(projectDir, constants.APP_FOLDER_NAME);
 					this.appResourcesDirectoryPath = path.join(projectDir, constants.APP_FOLDER_NAME, constants.APP_RESOURCES_FOLDER_NAME);
-					this.projectId = data.id;
+					this.projectIdentifiers = this.getProjectIdentifiers(data.id);
```

Comment:
please rename getProjectIdentifiers to something that better describes what the method does

Reply:
Not sure I understand what the naming of the method should be. I am open to suggestions.

Reply:
my suggestion is `initializeProjectIdentifiers`